scripts/monthly_stats.py

```diff
@@ -156,3 +172,18 @@ def send_email(self, email_address, email_body):
                                                                     'SPARC monthly dataset download summary',
                                                                     email_body)
 
+    def send_logging_email(self, message):
+        status_code = 000
+        try:
+            status_code = self.send_grid.sendgrid_email_with_unsubscribe_group(Config.METRICS_EMAIL_ADDRESS,
+                                                                 Config.METRICS_EMAIL_ADDRESS,
+                                                                 'SPARC monthly dataset download summary',
+                                                                 message)
+            if status_code != 202:
```

Comment:
Is there not a way to be more certain about the nature of the error?

Reply:
Yeah we could, it sends 403 for if the rate limit is hit. 401 if the key is wrong. I can add that in

Reply:
Have added this in now